### PR TITLE
Deploy censorship resistance to mainnet

### PR DESCRIPTION
**Pull request by ritave**

### CODE REVIEW

File: packages/contracts/truffle-config.js [hunk 1]
```diff
@@ -31,6 +39,21 @@ module.exports = {
       network_id: 4,
       gasPrice: "20000000000",
     },
+    ledgerMainnet: {
+      provider: function() {
+        var ledgerProvider = require("@joincivil/dev-utils").ledgerProvider;
+        var infura_key = process.env.INFURA_KEY;
+        var accountId = +process.env.LEDGER_ACCOUNT_ID;
```
Comment (walfly) [inline packages/contracts/truffle-config.js]: is this `+` right?

**Comment by nickreynolds** [comment]
I'm not sure we need the ledger provider stuff, but I guess it doesn't hurt

File: packages/contracts/truffle-config.js (hunk 1 above)
Comment (walfly) [reply packages/contracts/truffle-config.js]: @ritave

Comment (jorgelo) [reply packages/contracts/truffle-config.js]: @walfly, it looks correct, `accountId` needs to be a number and this is a tricky way to coerce.

**Comment by walfly** [comment]
Can't we just use Number class

**Pull request merged**